### PR TITLE
Builder design pattern

### PR DESCRIPTION
**Pull request by AhmedTuwaiq**
easy project

### CODE REVIEW

File: DesignPattern/WheatPizzaBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        public IPizzaBuilder BuildSauce()
+        {
+            this.Pizza.Add("Tomato sauce");
+            return this;
+        }
+
+        public IPizzaBuilder BuildTopping()
+        {
+            this.Pizza.Add("Cheese");
+            return this;
```
Comment (sarahAlqahtani) [inline DesignPattern/WheatPizzaBuilder.cs:39]: Here, you should call the reset method before returning the Pizza.

File: DesignPattern/WhitePizzaBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        public IPizzaBuilder BuildSauce()
+        {
+            this.Pizza.Add("Tomato sauce");
+            return this;
+        }
+
+        public IPizzaBuilder BuildTopping()
+        {
+            this.Pizza.Add("Cheese");
+            return this;
```
Comment (sarahAlqahtani) [inline DesignPattern/WhitePizzaBuilder.cs:39]: Here, you should call the reset method before returning the Pizza.

File: DesignPattern/Program.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        static void Main(string[] args)
+        {
+            WheatPizza wheatPizza;
+            WhitePizza whitePizza;
+
+            WheatPizzaBuilder wheatPizzaBuilder = new WheatPizzaBuilder();
+            WhitePizzaBuilder whitePizzaBuilder = new WhitePizzaBuilder();
+
+            Director wheatPizzaDirector = new Director(wheatPizzaBuilder);
```
Comment (SN-SAFCSP) [inline DesignPattern/Program.cs:15]: You don't need to declare another Director, one Director can work with any builder

File: DesignPattern/Program.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+namespace DesignPattern
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            WheatPizza wheatPizza;
+            WhitePizza whitePizza;
+
+            WheatPizzaBuilder wheatPizzaBuilder = new WheatPizzaBuilder();
```
Comment (SN-SAFCSP) [inline DesignPattern/Program.cs:12]: It would be better if you used polymorphism to define builders, without creating new builder variable for each one.

**Review by SN-SAFCSP — COMMENTED** [review]
Good job Ahmed!
Well done completing the project.
Here are some comments to improve your code:
Thank you for organizing your classes, and your code!

> don't forget to check comments on your code.

**Comment by SN-SAFCSP** [comment]
Good job Ahmed!
Well done completing the project.
Here are some comments to improve your code:
Thank you for organizing your classes, and your code!

> don't forget to check comments on your code.

**Pull request closed without merging**